__tests__/app.test.js

```diff
@@ -66,14 +66,13 @@ describe("app", () => {
             });
           });
       });
-      test("returns in date descending order", () => {
+      test("returns in date descending order, this shows the oldest and newest film", () => {
         return request(app)
           .get("/api/reviews")
           .expect(200)
           .then((res) => {
-            console.log(res.body.reviews, "Created at desc order");
             expect(res.body.reviews[0].created_at).toBe(
-              "2021-01-18T10:01:41.251Z"
+              "2021-01-25T11:16:54.963Z"
```

Comment:
This test can be done in a much simpler way using the jest-sorted library which i would recommend looking into.

Reply:
Completed with jest sorted